pages/common/carp.md

```diff
@@ -0,0 +1,28 @@
+# carp
+
+> REPL and build tool for Carp.
+> More information: <https://carp-lang.github.io/carp-docs/Manual.html>.
+
+- Start a REPL:
+
+`carp`
+
+- Start a REPL with a custom prompt:
+
+`carp --prompt "> "`
+
+- Build a file:
+
+`carp -b {{path/to/file.carp}}`
+
+- Build and run a file:
+
+`carp -x {{path/to/file.carp}}`
+
+- Build a file with optimizations:
+
+`carp -b --optimize {{path/to/file.carp}}`
+
+- Transpile a file to C code:
```

Comment:
And what will it do with it? Print it or save it to a file?

Reply:
As all build operations do, it saves to a file.